File: app/src/main/java/com/kidssaveocean/fatechanger/onboarding/OnboardingActivity.java
package com.kidssaveocean.fatechanger.onboarding;

import android.os.Bundle;
import android.support.design.widget.TabLayout;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;

import com.kidssaveocean.fatechanger.R;

import butterknife.BindView;
import butterknife.ButterKnife;

public class OnboardingActivity extends AppCompatActivity {

    @BindView(R.id.viewpager) ViewPager viewPager;
    @BindView(R.id.dot_indicator) TabLayout dotIndicator;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_onboarding);

//        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
//            Window w = getWindow();
//            w.setFlags(WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS, WindowManager.LayoutParams.FLAG_TRANSLUCENT_STATUS);
//        }

        ButterKnife.bind(this);

        viewPager.setAdapter(new OnboardingAdapter(getSupportFragmentManager()));
        dotIndicator.setupWithViewPager(viewPager);
    }

}

File: app/src/main/java/com/kidssaveocean/fatechanger/onboarding/OnboardingActivity.kt
package com.kidssaveocean.fatechanger.onboarding

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

import com.kidssaveocean.fatechanger.R

import kotlinx.android.synthetic.main.activity_onboarding.*

class OnboardingActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_onboarding)
        viewpager.adapter = OnboardingAdapter(supportFragmentManager)
        dot_indicator.setupWithViewPager(viewpager)
    }

}
